plugins/module_utils/common.py

```diff
@@ -551,6 +554,11 @@ def find_image_by_id(self, image_id):
         self.log("Find image %s (by ID)" % image_id)
         try:
             inspection = self.inspect_image(image_id)
+        except NotFound as exc:
+            if not accept_not_there:
```

Comment:
I always try to avoid variable names with `not` in them. This line illustrates a bit why: reading it aloud, it goes like "if not accept not there". Maybe `accept_missing_image` ?

Reply:
Fixed in 7f0ec90ac5b01f09bed827fece52e52e4f754c9e.